tests/test_microtiles.py

```
@@ -0,0 +1,462 @@
+
+    # Test no closure as well as ...
+    closure_directions = [None]
+    # ... every closure implemented
+    if tile_creator._closure_directions is not None:
+        closure_directions += tile_creator._closure_directions
+
+    # Retrieve shape values of parameters
+    n_eval_points = tile_creator._evaluation_points.shape[0]
+    n_info_per_eval_point = tile_creator._n_info_per_eval_point
+
+    def derivative_finite_difference_evaluation(
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `derivative_finite_difference_evaluation`

(ANN202)